electrum/gui/qt/staking_detail_tx_window.py

```diff
@@ -568,28 +579,32 @@ def insert_data(self, vbox):
 
         hbox_status = QHBoxLayout()
         status_lab = QLabel('Status:')
-        status_lab_data = QLabel('Stake')
+        status_lab_data = QLabel('Claimed')
```

Comment:
use translation string _(xx)

also, at some point we need to review all new widgets in terms of translatability